QEfficient/generation/text_generation_inference.py

```diff
@@ -651,15 +674,17 @@ def run_decode(self, decode_inputs, generation_len):
             num_token (int): The number of tokens processed in the decoding process.
         """
         finished_sequences = decode_inputs["input_ids"] == self.tokenizer.eos_token_id
+        if stream:
```

Comment:
Fixed the issue by creating a streamer object within TextGeneration class (_regular_model_execution method) if stream mode is set and streamer obj is not yet instantiated. run_decode will take streamer obj as input instead of stream boolean value.